### PR TITLE
WIP: JuliaRequirement: Add installation methods

### PR DESCRIPTION
**Pull request by RohanVB**

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
but it will be merged eventually. so please fix the coala problems and stuff, and if you dont want it to be reviewed by other people, mark the Pull Request title starting with ``WIP:``

**Comment by RohanVB** [comment]
Add: Installation methods for JuliaRequirement

JuliaRequirement could have the install_command method
 which builds the command from its attributes,
and the is_installed method which checks
 whether it is installed or not.

Fixes https://github.com/coala-analyzer/coala/pull/2435


^ this is my commit message. why is it saying Head > 50? Head = 45.

File: .misc/deps.sh [hunk 1]
```diff
@@ -10,12 +10,15 @@ case $CIRCLE_NODE_INDEX in
 esac
 
 # apt-get commands
+sudo add-apt-repository ppa:staticfloat/juliareleases
+sudo add-apt-repository ppa:staticfloat/julia-deps
```
Comment (arafsheikh) [inline .misc/deps.sh]: `sudo add-apt-repository -y ...` is needed

File: coalib/bears/requirements/JuliaRequirement.py [hunk 2]
```diff
@@ -29,3 +30,24 @@ def __init__(self, package, version="", flag=""):
         """
         PackageRequirement.__init__(self, 'julia', package, version)
         self.flag = flag
+
+    def install_command(self):
+        """
+        Creates the installation command for the instance of the class.
+
+        >>> JuliaRequirement("Lint").install_command()
+        'julia -e "Pkg.add(\"Lint\")'
```
Comment (arafsheikh) [inline coalib/bears/requirements/JuliaRequirement.py]: To get the expected output you need `JuliaRequirement("Lint", flag="-e").install_command()`

**Comment by arafsheikh** [comment]
43e5508 needs work

File: coalib/bears/requirements/JuliaRequirement.py (hunk 2 above)
Comment (Adrianzatreanu) [inline coalib/bears/requirements/JuliaRequirement.py]: Theres bo more flag, i removed it in my merged PR. He needs to pull though

**Comment by RohanVB** [comment]
If anyone can take over this PR, that'd be great.
Ping me if you need any help.

**Comment by Makman2** [comment]
Taking over here: https://github.com/coala-analyzer/coala/pull/2732
:)

**Pull request closed without merging**